### PR TITLE
Refacto ui of GeometryField

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -106,14 +106,15 @@
...
+    "button": "File ({{accept}})",
     "intro1": "When selecting the file, it is analyzed by the application. You will then be informed about the compatibility of the file with the current geometry type.",
     "intro2": "Upon validation of the import, the application offers on the map an overview of the new geometry.",
     "intro3": "Upon validation of this new imported sketch, it replaces the previous one.",
     "nothingToImport": "No one geometry from {{fileName}} could be imported.",
     "readyToImport": "{{fileName}} geometry is ready to be imported.",
     "readyToImport_plural": "{{fileName}} geometries are ready to be imported.",
     "submit": "Confirm selection",
-    "title": "Import a geometry file",
+    "title": "Import coodinates (WGS84 Décimal)",
```
Comment (jrmi) [inline public/locales/en/translation.json:117]:
```suggestion
    "title": "Import coordinates (WGS84 Décimal)",
```

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -336,13 +336,15 @@
...
-      "label": "Enter coordinates:",
+      "label": "Or enter coordinates",
+      "legend": "Coordinates",
       "title": "Coordinates (decimal WGS84)",
       "latitude": "Latitude",
       "longitude": "Longitude",
-      "helper-create": "Draw $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
-      "helper-edit": "Edit $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
-      "save": "Once edited, click on submit button."
+      "information-create": "Or draw $t(jsonSchema.geometryField.type.{{type}}.singular) on map.",
```
Comment (jrmi) [inline public/locales/en/translation.json:344]: Can you simplify the interpolate var ?

File: src/components/react-json-schemaForm/GeometryField/Informations.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import React, { memo } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import Message from '../../Message';
+
+function getIconFromType (type) {
```
Comment (jrmi) [inline src/components/react-json-schemaForm/GeometryField/Informations.js:6]: Why not an arrow function ?

File: public/locales/en/translation.json (hunk 2 above)
Comment (dtrucs) [reply public/locales/en/translation.json:344]: What do you think? 
According to `type`, it should pick the correct wording between lines 317 and 337

File: public/locales/en/translation.json (hunk 1 above)
Comment (dtrucs) [reply public/locales/en/translation.json]: Fixed 👍

File: src/components/react-json-schemaForm/GeometryField/Informations.js (hunk 3 above)
Comment (dtrucs) [reply src/components/react-json-schemaForm/GeometryField/Informations.js]: Dunno but fixed

**Review by jrmi — APPROVED** [review]

**Pull request merged**